awsbestpractices/mandatory-method/MandatoryMethod.java

```diff
@@ -0,0 +1,31 @@
+import javax.swing.ImageIcon;
+import com.amazonaws.services.simplesystemsmanagement.model.PutParameterRequest;
+
+public class MandatoryMethod {
+
+    public void checkWindowDescription() {
+        //Should not flag
+        CreateMaintenanceWindowRequest maintainanceWindow = new CreateMaintenanceWindowRequest();
+        maintainanceWindow.setDescription("Description");
+
+        //Should not flag
+        CreateMaintenanceWindowRequest maintainanceWindow2 = new CreateMaintenanceWindowRequest();
+        maintainanceWindow2.withDescription("Description");
+
+        //Should flagS
+        CreateMaintenanceWindowRequest maintainanceWindow3 = new CreateMaintenanceWindowRequest();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Consider [specifying a description](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/simplesystemsmanagement/model/CreateMaintenanceWindowRequest.html#setDescription-java.lang.String-) to help you organize your maintenance windows.